### DISCUSSION TITLE
New master and slave option

### DISCUSSION THREAD

**Discussion by MicheleVilla** [EdgeTX/edgetx#4581] [Ideas]
When using a master and slave radio the model parameters are stored on the master radio and the slave is used only as a command
It often happens that there is an instructor who has multiple students using their radios already associated with their models
it may be useful to be able to configure the master radio to be used only as a command with the master slave switch and the slave radio associated with the model with all parameters and the active transmitter module
in this case the instructor does not have to copy all the students' programs onto his radio, but simply connect it to the student's radio

I hope I was clear

**Comment by MicheleVilla** [comment]
Back to this idea
A possible problem could occur when the trainer cable is disconnected by mistake and the control of the model remains on the student radio.
for the rest, if feasible, it would be very useful to simplify the management of the students

**Comment by ChrisOhara57** [comment]
I hope I do not misunderstand your proposal.
But I think what you want is nothing else as spotter configuration.
You need an extra channel to be send from the slave (trainer) radio to the master (student) radio.
With this channel the master radio is told whether to switch to trainer or student.

I use this with an old Mulitplex Cockpit MM radio as trainer  and different student Radios (x9lite, X7 and X9D).

I have also uploaded a description  (for both trainer and spotter configuration) in issue
"Trainer settings per model and not system wide #4916"
